Lib/ipaddress.py

```diff
@@ -1511,7 +1512,6 @@ def __init__(self, address, strict=True):
             self.hosts = self.__iter__
 
     @property
-    @functools.lru_cache()
```

Comment:
Why did you removed this decorator?

Reply:
because self.network_address.is_global which in turn calls is_private already has a cache. There are pros and cons for example if someone removes cache there then suddenly this would loose caching behavior and also this having its own cache frees up the other cache. Should I add it back ?

Reply:
because self.network_address.is_global which in turn calls is_private already has a cache. There are pros and cons for example if someone removes cache there then suddenly this would loose caching behavior and also this having its own cache frees up the other cache. Should I add it back ?